kernel/content/ezcontentoperationcollection.php

```diff
@@ -866,6 +870,11 @@ static public function removeNodes( array $removeNodeIdList )
 
             if ( !isset( $objectIdList[$objectId] ) )
                 $objectIdList[$objectId] = eZContentObject::fetch( $objectId );
+            eZAudit::writeAudit( 'location-remove', array( 'Parent Node ID' => $node->attribute( 'parent_node_id' ),
+                                              'Node ID' => $nodeId,
```

Comment:
Just a note here, this function is a mess and I decided to log each node individually, but yes, we can provide the information about the main node, currently the Node ID refers to the removed node.